src/lightning_app/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+## \[0.5.2\] - 2022-MM-DD
+
+### Added
+
+- Update the Lightning App docs ([#13537](https://github.com/PyTorchLightning/pytorch-lightning/pull/13537))
```

Comment:
This wouldn't be a CHANGELOG addition

Reply:
This is almost the entire documentation. I believed it can be added as we have an empty changelog and the PR has 7k lines.